webui/react/src/components/Table/InteractiveTable.tsx

```diff
@@ -305,19 +309,22 @@ const HeaderCell = ({
       <div
         className={`${className} ${css.columnDraggingDiv}`}
         ref={drag}
-        title={columnName}
+        title={unusedTitleFromAntd as string}
         onClick={(e) => e.stopPropagation()}
         {...props}
       />
       <DraggableCore
         nodeRef={resizingRef}
         onDrag={(e, data) => {
-          const minWidth = onResize(e, data);
+          onResize(e, data);
           const newWidth = data.x < minWidth ? minWidth : data.x;
 
           if (newWidth !== xValue) setXValue(newWidth);
         }}
         onStart={(e, data) => {
+          setShadowVisibility('block');
+          const newWidth = data.x < minWidth ? minWidth : data.x;
+          if (newWidth !== xValue) setXValue(newWidth);
```

Comment:
the changes in this component are to fix the bug (currently in master) where the resizer is in the wrong place at the start

Uploading Screen Recording 2022-10-12 at 3.09.37 AM.mov…

Reply:
Can you put this in its own PR since it's unrelated to this?

Reply:
will do.